### DISCUSSION TITLE
下载获取链接失败

### DISCUSSION THREAD

**Discussion by asd119** [the1812/Bilibili-Evolved#3015] [提问 / Q&A] [answered]
链接请求被拦截，该如何解决
![image](https://user-images.githubusercontent.com/97240596/154226553-f8094765-211f-44d0-bbcd-53c26fbd1cb8.png)

**Comment by the1812** [comment, marked as answer]
短时间内下载太多了？

**Reply by asd119** [reply]
emmmm，是的

**Reply by the1812** [reply]
那就是被 b 站临时封禁了，你大概下了多少

**Reply by asd119** [reply]
大概5个g的视频

**Reply by asd119** [reply]
一次性的。大概啥时候能解

**Reply by the1812** [reply]
视频数量多少，不是大小

解的话我不清楚，反正得等一段时间

**Reply by asd119** [reply]
97个
